FlipperZero/Core/Container/ObservableResolver.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ObservableResolver.swift
+//  FlipperZero
+//
+//  Created by Eugene Berdnikov on 8/23/20.
+//
+
+import Combine
+
+public class ObservableResolver: Resolver, ObservableObject {
+    private let container: Container
+
+    public init() {
+        self.container = Container()
+        self.container.register(BluetoothService.init, as: BluetoothConnector.self, isSingleton: true)
```

Comment:
Maybe it's better to rename `ObservableResolver` to `ObservableBTServiceResolver`? Or if this class is created with intention of also resolving different types of services maybe it is better to expect an enum case as init parameter? E.g.:
```swift
public init(_ type: ServiceType){
    self.container = Container()
    switch type {
    case .bluetooth:
        self.container.register(BluetoothService.init, as: BluetoothConnector.self, isSingleton: true)
    // TODO: add cases for different services when needed.
    default: break
    }
}
```

Reply:
`ObservableResolver` is a wrapper for `Container`. We need it because container is used with the `@StateObject` attribute, so it should subclass the `ObservableObject`.
In addition, we need a place where we'll register all the dependencies (unless we use some framework which allows implicit registration), so `ObservableResolver` looks like a good candidate for it (I'm open to a different approach though)
We don't need to create more than one instance of `ObservableResolver`

Reply:
Oh, I see. Multiple service types would be registered on `ObservableResolver` initialization in a single container, rather then creating additional instances. I made a wrong assumption looking at `init`.